NuGet.config

```diff
@@ -19,6 +19,9 @@
     <!-- TODO: Remove dotnet7 feeds when dependencies publish into dotnet8 feeds: https://github.com/dotnet/runtime/issues/63375. -->
     <add key="dotnet7" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json" />
     <add key="dotnet7-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7-transport/nuget/v3/index.json" />
+    <!-- TODO: Remove dotnet8 feeds when dependencies publish into dotnet9 feeds: https://github.com/dotnet/runtime/issues/74100 -->
```

Comment:
That note isn't necessary. This will naturally happen in a year when we brand to .NET 9.

```suggestion
```